src/include/distributed/backend_data.h

```diff
@@ -21,6 +22,9 @@
 #include "storage/s_lock.h"
 
 
+#define TotalProcs MaxBackends + NUM_AUXILIARY_PROCS + max_prepared_xacts
```

Comment:
Maybe put the expression into parentheses. That avoids bugs like `TotalProcs * 2` getting translated to `MaxBackends + NUM_AUXILIARY_PROCS + max_prepared_xacts * 2`

Reply:
Oops, that's a nasty bug and already the case on `BuildLocalWaitGraph ()` (though not important at that point).